core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -319,14 +319,24 @@ default void subscribe(Subscriber<? super T> subscriber, SubscriptionOption... o
     CompletableFuture<List<T>> drainAll(EventExecutor executor, SubscriptionOption... options);
 
     /**
-     * To duplicator.
+     * Returns a new {@link StreamMessageDuplicator} that duplicates multiple {@link StreamMessage}s which
+     * publish the same elements with this {@link StreamMessage}.
```

Comment:
`that duplicates this {@link StreamMessage} into one ore more {@link StreamMessage}s, which publish the same elements.` ?

Reply:
Fixed. 😉